service/ziti-tunnel/service/state.go

```diff
@@ -40,9 +40,15 @@ import (
 	"path"
 	"strconv"
 	"strings"
+	"sync/atomic"
 	"time"
 )
 
+var tun_release atomic.Value
```

Comment:
Probably would call this the "state" or "status", not so much "release". I'd probably have put this into the runtime state struct. We there a reason you couldn't do that?